Calculator/CalculatorViewController.swift

```diff
@@ -12,31 +12,47 @@ class CalculatorViewController: UIViewController {
     
     @IBOutlet weak var outputLabel: UILabel!
     
+    var brain: CalculatorBrain?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        brain = CalculatorBrain()
     }
     
     // MARK: - Action Handlers
     
     @IBAction func operandTapped(_ sender: UIButton) {
-        
+        if let buttonTitle = sender.currentTitle {
```

Comment:
`.currentTitle` is for buttons whose title changes from state to state. In a weird way, the sender in this calculator app is kind of doing the same thing. SO, IN THIS CASE, using currentTitle could make sense. 

But the correct way to access the title for this button is: 
`sender.titleLabel?.text`